content/the-ember-times-issue-173.md

```diff
@@ -36,14 +38,19 @@ Check out the full [plan for the 3.0.0 release](https://github.com/ember-templat
 
 ---
 
-## [3. Section title in sentence case 🐹](section-url)
+## Learn more Ember.js in 2021 💡
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+If you are new to Ember.js in 2021—first of all, on behalf of the broader community, a warm welcome from The Ember Times! We're lucky to have you and hope that you will say hello and ask questions. 👋🏼
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Over the past couple of months, several people have helped out with your learning experience by publishing blog posts and videos. We'd like to list them here for your convenience (apologies if we missed something; please let us know on `#support-ember-times` channel on [Discord](https://discord.gg/emberjs)):
```

Comment:
```suggestion
Over the past couple of months, several people have helped out with your learning experience by publishing blog posts and videos. We'd like to list them here for your convenience (apologies if we missed something; please let us know on the [`#support-ember-times`](https://discord.com/channels/480462759797063690/485450546887786506) channel on [Discord](https://discord.gg/emberjs)):
```

i only suggest this, cause i think some ppl still don't really understand discord with the # of channels, being in every channel and then having to mute them... kinda different from slack / maybe different from teams(?)

Reply:
Yep, thanks for adding the link!